src/js/providers/html5.js

```diff
@@ -11,8 +11,8 @@ define([
 ], function(cssUtils, utils, stretchUtils, _, events, states, eventdispatcher, DefaultProvider, video) {
 
     var clearInterval = window.clearInterval,
-        stallInterval,
         STALL_DELAY = 256,
+        BUFFER_INTERVAL = 50,
```

Comment:
I agree that more info is better, I just brought it up because the ticket was originally labeled "optimize-performance"